client/app.jsx

```diff
@@ -22,19 +29,35 @@ export default class App extends React.Component {
       .catch(err => console.error('GET error', err.message));
   }
 
-  render() {
+  renderPage() {
+    const prodId = Number(this.state.route.id);
+    const { route } = this.state;
     const products = this.state.products;
+
+    if (route.path === '') {
+      return (
+      <div className='row'>
+      {
+    products.reverse().map((product, index) => {
+      return <NewItems key={index} product={product} />;
```

Comment:
I noticed that this component is named `NewItems`, but it only shows one item. You should make a component named `NewItems` that actually takes the whole array and does mapping inside of itself. That will keep your `renderPage()` method much cleaner.